src/components/Checkout/OrderSummary.js

```diff
@@ -1,178 +1,209 @@
-            </Link>
-          ) : (
-            <Button
-              disabled={isLoading || !stripe || !elements}
-              size='md'
-              w='full'
-              bg={colors.primary}
-              color='white'
-              onClick={handleSubmit}
-            >
-              <span>
-                {isLoading ? (
-                  <div className='spinner' id='spinner'></div>
-                ) : (
-                  'Place Order'
-                )}
-              </span>
-            </Button>
-          )}
-        </GridItem>
-        <GridItem colSpan={2}>
-          {message && (
-            <div
-              style={{
-                color: 'rgb(105, 115, 134)',
-                fontSize: '16px',
-                lineHeight: '20px',
-                paddingTop: '12px',
-                textAlign: 'center',
-              }}
-            >
-              {message}
-            </div>
-          )}
-        </GridItem>
-      </SimpleGrid>
-    </VStack>
-  );
+        setIsLoading(false);
+    };
+
+    useEffect(() => {
+        if (!stripe) {
+            return;
+        }
+
+        const clientSecret = new URLSearchParams(window.location.search).get(
+            'payment_intent_client_secret'
+        );
+
+        if (!clientSecret) {
+            return;
+        }
+
+        stripe.retrievePaymentIntent(clientSecret).then(({ paymentIntent }) => {
+            switch (paymentIntent.status) {
+                case 'succeeded': {
+                    toast({
+                        title: 'Payment succeeded.',
+                        status: 'success',
+                        duration: 9000,
+                        isClosable: true,
+                    });
+                    break;
+                }
+
+                case 'processing': {
+                    toast({
+                        title: 'Payment processing.',
+                        status: 'info',
+                        duration: 9000,
+                        isClosable: true,
+                    });
+                    break;
+                }
+                case 'requires_payment_method': {
+                    toast({
+                        title: 'Payment unsuccessful.',
+                        status: 'error',
+                        duration: 9000,
+                        isClosable: true,
+                    });
+                    break;
+                }
+                default: {
+                    toast({
+                        title: 'Server error',
+                        status: 'error',
+                        duration: 9000,
+                        isClosable: true,
+                    });
+                    break;
+                }
+            }
+        });
+    }, [stripe]);
+
+    return (
+        <VStack
+            w='full'
+            p={10}
+            spacing={10}
+            alignItems='flex-start'
+            style={{
+                border: '2px solid #E2E8F0',
```

Comment:
sorry forgot can you change border to 1 px

Reply:
done